web/styles/zulip.css

```diff
@@ -3295,3 +3295,86 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#add-poll-modal {
+    /* on wide screens, this height allows roughly 3 option
+    rows to fit in without need for scrolling */
+    height: 450px;
+    overflow: hidden;
+
+    @media (width < $sm_min) {
+        height: 80vh;
+    }
+
+    .modal__content {
+        flex-grow: 1;
+
+        .simplebar-content {
+            box-sizing: border-box;
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+        }
+    }
+
+    #add-poll-form {
+        display: flex;
+        flex-direction: column;
+        overflow: hidden;
+        height: 100%;
+
+        & label {
+            font-weight: bold;
+            margin: 5px 0;
+        }
+
+        & input[type="text"] {
+            border: 1px solid hsl(0deg 0% 0% / 20%);
+            outline: 0;
+            height: 1.8rem;
+        }
+
+        #poll-question-input {
+            margin-bottom: 10px;
+        }
+
+        .poll-options-list {
+            margin: 0;
+            height: 0;
+            overflow: auto;
+            flex-grow: 1;
+
+            .option-row {
+                list-style-type: none;
+                cursor: move;
+                margin-top: 10px;
+                padding: 0;
+
+                .fa-ellipsis-v {
+                    color: hsl(0deg 0% 75%);
+                    position: relative;
+                    top: 1px;
+                }
+
+                /* Space the right side ellipsis from the input next to it */
+                & i + i {
+                    margin-right: 5px;
+                }
+
+                & input {
+                    width: clamp(190px, 50%, 300px);
+                }
+            }
+
+            .option-row:first-child {
+                margin-top: 0;
```

Comment:
Why not just use `margin-bottom` to separate these? It's probably fine to have some extra margin at the bottom, and then you can save yourself this first-child selector.

Reply:
I did consider using `margin-bottom`, however unless we remove it from the last row (which is analogous to this current code) it tends to cause an unnecessary scrollbar when there are 4 options, since all for rows fit visually, but the last option's bottom margin doesn't and causes the scrollbar to appear